src/pages/billing/Project.js

```diff
@@ -662,10 +702,8 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
   useEffect(() => { StateHistory.update({ projectUsers }) }, [projectUsers])
   // Update cost data only if report date range changes, or if spend report tab was selected.
   useEffect(() => {
-    const maybeLoadProjectCost = _.flow(
-      reportErrorAndRethrow('Unable to retrieve spend report data'),
-      Utils.withBusyState(setUpdating)
-    )(async () => {
+    const maybeLoadProjectCost = async () => {
```

Comment:
I believe you can still use `Utils.withBusyState` to handling the toggling of `setUpdating`.